src/components/VisualizeLinear/CorrelationDescrip/CorrelationDescrip.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './CorrelationDescrip.css'
+import PositiveCorrelation from '../../../assets/VisualizeLinear/PositiveCorrelation.svg'
+import NegativeCorrelation from '../../../assets/VisualizeLinear/NegativeCorrelation.svg'
+import Box from '@material-ui/core/Box';
+import { Typography } from "@material-ui/core";
+import CorrelationModal from '../CorrelationModal/CorrelationModal'
+import Button from '@material-ui/core/Button';
+
+const CorrelationDescrip = () => {
+    const [open, setOpen] = React.useState(0);
+
+    const handleOpenA = () => {
+        setOpen(1)
+    }
+
+    const handleOpenB = () => {
+        setOpen(2)
+    }
+
+    const handleClose = () => {
+        setOpen(0)
+    }
+    // const [openA, setOpenA] = React.useState(0);
+    // const [openB, setOpenB] = React.useState(0);
+    // console.log("CorrelationModal")
+  
+    // const handleOpenA = () => {
+    //   setOpenA(true);
+    // };
+
+    // const handleOpenB = () => {
+    //     setOpenB(true);
+    // };
+    
+    // const handleCloseA = () => {
+    //   setOpenA(false);
+    // };
+    
+    // const handleCloseB = () => {
+    //     setOpenB(false);
+    // };
+
+    return(
+        <div className = "CorrelationDescrip">
```

Comment:
We should wrap this in a Card, like we did for the introlinear page.

Reply:
I think if we wrap this in a card it will only wrap the text and 2 graphs, and not the data table on the right hand side.  I'm not really sure if that's what we wanted for this section?